flowmachine/flowmachine/features/utilities/event_table_subset.py

```diff
@@ -164,7 +173,47 @@ def _check_dates(self):
                 stacklevel=2,
             )
 
-    def _make_query(self):
+    def _make_query_with_sqlalchemy(self):
+        table = get_sqlalchemy_table_definition(
```

Comment:
Should we call `sqlalchemy_table` to match with `sqlalchemy_columns`?

Reply:
👍